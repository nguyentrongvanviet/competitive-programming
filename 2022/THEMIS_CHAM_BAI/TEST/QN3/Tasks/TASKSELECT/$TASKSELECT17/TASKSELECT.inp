10
1 384531527
1 617611958
1 475044389
1 918179178
1 100156401
2 323257557
2 743794732
3 912288949
2 21108959
3 1000000000
