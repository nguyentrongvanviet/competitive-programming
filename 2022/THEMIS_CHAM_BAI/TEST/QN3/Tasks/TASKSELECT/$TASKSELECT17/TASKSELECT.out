1
1
1
1
1
161628778
371897365
263991094
10554479
250000010
